client/src/pages/OrderScreen.js

```diff
@@ -12,7 +12,7 @@ import TextIcon from "components/TextIcon";
 import TopTitleBar from "components/TopTitleBar";
 
 const serverURL = "http://localhost:9000";
-const menuId = "6103677a11c316178047f1f1";
+const DEFAULT_MENU_ID = "6103677a11c316178047f1f1";
```

Comment:
will we be keeping this hard coded or changing it in the future

Reply:
hard coded unless we change the default menu

Reply:
why is there a hard coded default menu? shouldn't we pull from the db the list of menus by Thursday?

Reply:
cause technically it's supposed to be production ready